### PR TITLE
Simple setup docs and package name override to avoid conflicts when necessary

### PR DESCRIPTION
**Pull request by cjolivier01**
Simple setup docs and package name override to avoid conflicts when necessary

### CODE REVIEW

**Review by Zenix27 — APPROVED** [review]

**Review by ailzhang — APPROVED** [review]
Thanks! LGTM!

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+#
+#   TORCH_XLA_VERSION
+#     specify the version of PyTorch/XLA, rather than the hard-coded version
+#     in this file; used when we're building binaries for distribution
+#
+#   VERSIONED_XLA_BUILD
+#     creates a versioned build
+#
+#   TORCH_XLA_PACKAGE_NAME
+#     change the package name to something ther than 'torch_xla'
```
Comment (JackCaoG) [inline setup.py:17]: "other than"

**Review by JackCaoG — COMMENTED** [review]
one minor typo

File: setup.py (hunk 1 above)
Comment (cjolivier01) [reply setup.py]: done

**Review by JackCaoG — APPROVED** [review]
Thanks @cjolivier01 !

**Pull request merged**